assert(1 > 2, "This assertion failed");